family_foto/templates/gallery.html

```diff
@@ -2,9 +2,9 @@
 
 {% block content %}
     <h1>Gallery </h1>
-    {% for medium in media %}
+    {% for medium, thumbnail in media %}
         <a href="{{ medium.image_view }}">
-            <img src="{{ medium.thumbnail(200,200) }}" alt="{{ medium.filename }}"
+            <img src="{{ thumbnail }}" alt="{{ medium.filename }}"
```

Comment:
maybe change the alt text to something else so people dont get confused

Reply:
Like an error message?